src/main/java/com/cburch/logisim/std/memory/Ram.java

```diff
@@ -381,6 +389,16 @@ private void propagateByteEnables(InstanceState state, long addr, boolean goodAd
     }
   }
 
+  private Value invertValue(Value input) {
```

Comment:
Yeah, I originally made this method with the intention of avoiding having an ERROR `weValue` if the input is unknown, but upon further inspection I see that it doesn't really matter since it only ever gets compared to TRUE and FALSE, I'll go with .not() instead. Thanks for pointing it out